### PR TITLE
(#171) Added flake8 options and added to CI

### LINKED ISSUE #171: Flake8: Fix errors and run in CI
Currently we have many flake8 warnings, we should fix them and add checks in CI

### PR DESCRIPTION
**Pull request by DominicOram**
Fixes #171

### CODE REVIEW

File: src/artemis/devices/zebra.py [hunk 1]
```diff
@@ -74,14 +75,16 @@ def arm_status(self, armed: int) -> StatusBase:
 class ZebraOutputPanel(Device):
     pulse_1_input: EpicsSignal = epics_signal_put_wait("PULSE1_INP")
 
-    out_1: EpicsSignal = epics_signal_put_wait(f"OUT1_TTL")
-    out_2: EpicsSignal = epics_signal_put_wait(f"OUT2_TTL")
-    out_3: EpicsSignal = epics_signal_put_wait(f"OUT3_TTL")
-    out_4: EpicsSignal = epics_signal_put_wait(f"OUT4_TTL")
+    out_1: EpicsSignal = epics_signal_put_wait("OUT1_TTL")
```
Comment (abbiemery) [inline src/artemis/devices/zebra.py:78]: Were these just unused fstrings?

**Review by abbiemery — COMMENTED** [review]
For the files that have been changed, they're mostly import changes. I'm assuming these are removing un-used imports and do not affect the code?

File: src/artemis/devices/zebra.py (hunk 1 above)
Comment (MattPrit) [reply src/artemis/devices/zebra.py:78]: Yes

**Comment by DominicOram** [comment]
I can't approve this as it's technically my own PR but mostly it's @MattPrit's work. I'm happy with it.

**Review by abbiemery — COMMENTED** [review]
Nothing looks particularly out of order, but the code cov patch now fails. Maybe sort that first. @MattPrit

**Review by MattPrit — APPROVED** [review]

**Pull request merged**